src/org/omegat/gui/preferences/view/EditingBehaviorController.java

```diff
@@ -72,10 +72,10 @@ private void initGui() {
 
     @Override
     protected void initFromPrefs() {
-        panel.defaultRadio.setSelected(!Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
-        panel.leaveEmptyRadio.setSelected(Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
+        panel.defaultRadio.setSelected(Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
+        panel.leaveEmptyRadio.setSelected(!Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT));
```

Comment:
It looks better to introduce a `Prefrences#isNotPreference` method other than calling `!Preferences.isPrefernce` for better recognition of logics.

Reply:
I understand. See my comment above.

I have 2 concerns now:

1. The regex I set is not taken into account by default in OmegaT even though it is displayed, there is a need to "reset the defaults" to have OmegaT take it into account.
2. The whole set of preferences needs an overhaul, see comment above, but I'd rather work on that after the release, to not introduce more delays.

Reply:
I can't quite follow how the set of preferences works either, but based on my brief testing:
1. The above block sets the radio button in the Preferences dialog to "Leave the dialog empty", but it does not actually set the preference: the source segment is still inserted automatically when I enter a new segment. The `prefs` file generated by this version of the does not contain the "<wf_noSourceText>true</wf_noSourceText>" entry I get in the `prefs` file when I set the preference in a build made from master. (I couldn't figure out where to set it, though.)

2. If anyone is willing to explain to me how the current system works, I'd be happy to try to help with this after the release.